dev-docs/RFCs/README.md

```diff
@@ -22,12 +22,17 @@ Writeups of directions in major areas of interest
 | [**Loader Lookup By Name**](vNext/loader-lookup-by-name-rfc.md)       | @ibgreen | **Draft** | Loader lookup among pre-registered loaders                  |
 | [**Loader Auto-Registration**](vNext/loader-auto-registration-rfc.md) | @ibgreen | **Draft** | Loader auto registration at import                          |
 
+## v2.3 RFCs
```

Comment:
What release is this targeting? This is a breaking change. Existing applications that provide load options have to supply fetch options at root, not the nested object.

Reply:
Yes you are right. I'll update to make sure we still provide root `options` to fetch if `options.fetch` is not provided until the next major release, and I can add some tests with a mock fetch function to make sure this works for both cases.

But I'd really like to phase out such unstructured uses of prop objects, so I'll add a comment stating that root object support is deprecated and should be removed in next major release.

I'm also interested in your take on whether this can be designed in such a way that we can phase out custom `fetch` props in the deck.gl API (e.g. in deck.gl v9). Handling of custom fetch in deck.gl seems like something that would be great to delegate to loaders.gl, and the changes we make here will be exposed through the deck `loaderOptions` prop. Happy to consider additional changes to make that possible.

Reply:
I’m in for removing `fetch` prop from deck.

Regarding backward compatibility, I strongly recommend checking the root object for common `fetch` options (mode, headers, credentials etc.) and issue a warning before we completely drop the support.

Reply:
> I strongly recommend checking the root object for common fetch options (mode, headers, credentials etc.) and issue a warning before we completely drop the support.

Yup, exactly my thought, and also fits well with the new loader option validation system.